examples/3d-tiles/tile-3d-layer.js

```diff
@@ -261,10 +261,8 @@ export default class Tile3DLayer extends CompositeLayer {
           this._unpackPointCloud3DTile(tileHeader);
           break;
         case 'i3dm':
-          this._unpackInstanced3DTile(tileHeader);
```

Comment:
I recommend keeping separate unpack functions to unpack into common data structure, rather than add on more if statements in the render loop.

The differences between the two formats is big enough that sharing a function does not make sense in my opinion.